internal/armada/repository/usage.go

```diff
@@ -75,43 +76,58 @@ func (r *RedisUsageRepository) GetClusterLeasedReports() (map[string]*api.Cluste
 func (r *RedisUsageRepository) GetClusterPriority(clusterId string) (map[string]float64, error) {
 	result, err := r.db.HGetAll(clusterPrioritiesPrefix + clusterId).Result()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("[RedisUsageRepository.GetClusterPriority] error reading from database: %s", err)
 	}
-	return toFloat64Map(result)
```

Comment:
What's the story behind toFloat64Map returning an error as well? Did it always do this and we just ignored the error? Is this a thing that can commonly fail?

Reply:
It parses strings to floats, which can fail. I think that before we just didn't wrap it, i.e., the caller had to handle the underlying error directly.